Packs/Gmail/Integrations/Gmail/README.md

```diff
@@ -327,6 +327,16 @@
           <li>
             <a href="#h_426da8de-404c-432d-9080-b476cf896g9a" target="_self">Get details of a specific role: gmail-get-role</a>
           </li>
+           <li>
```

Comment:
Please refactor the readme to the new format , please use `demisto-sdk generate-docs` 
You can see some more examples:
https://xsoar.pan.dev/docs/integrations/integration-docs
Make sure your additional command are added